blockchain/contracts/ElectionFactory.sol

```diff
@@ -0,0 +1,76 @@
+// SPDX-License-Identifier: MIT 
+pragma solidity ^0.8.24;
+
+import "./Election.sol";
+
+contract ElectionFactory {
+  address public owner;
+  //Array ti keep track of the elections
+  address[] public elections;
```

Comment:
@theGeniusDennis Let also not make the elections public since there is a `getElections()` functions that is public.